ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DOWNLINK_IPV4_POOL_TESTS.yml

```diff
@@ -0,0 +1,48 @@
+default_node_types:
+  - node_type: spine
+    match_hostnames:
+      - .*spine\d
+  - node_type: l3leaf
+    match_hostnames:
+      - .*l3leaf\d
+
+spine:
+  defaults:
+    loopback_ipv4_pool: 192.168.0.0/24
+    bgp_as: 65000
+  nodes:
+    - name: downlink-ipv4-pool-spine1
+      downlink_pools:
+        - downlink_ipv4_pool: 10.0.1.0/24
+          downlink_interfaces: [Ethernet3, Ethernet4]
```

Comment:
Try adding multiple uplinks to the same uplink_switch.

Reply:
I don't quite understand what you mean here, I have added 2 downlink_ipv4_pool(s) to spine2, is that not what you mean?

Reply:
You do not have any leaf with more than 1 link to each spine. Please add that.